### PR TITLE
Initial draft

### PR DESCRIPTION
**Pull request by samdark**
- Initial text is taken from PSR-12
- Meta and intro were adjusted to mention PER
- Some typos (commas mainly) were fixed

<strike>Do not merge before #2</strike>

### CODE REVIEW

File: meta.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+[here for finally blocks](https://gist.github.com/michaelcullum/c025f3870c9ea1dd2668#file-finallyblocks-php)
+and the current implementation was chosen due to consistency with other parts of the standard.
+
+### 4.3. Enforcing short form for all type keywords
+
+PHP 7.0 introduced [scalar types declaration](http://php.net/manual/en/functions.arguments.php#functions.arguments.type-declaration)
+which does not support long type aliases. Therefore, it makes sense to enforce primary short type forms to be used to
+have uniform syntax and prevent possible confusion.
+
+### 4.4. Public Survey
```
Comment (Jean85) [inline meta.md:87]: I would split this and other too-old-stuff coming from PSR-2 in a separate file... But maybe we can do it later, when we make changes to the PER.

**Comment by KorvinSzanto** [comment]
I think it's important that we capture the current state of PSR-12 in an initial commit before we start modifying it so that we can track on those changes more easily. We should be able to get that merged in before merging this one and have everything work as expected.

File: README.md [hunk 2]
```diff
@@ -1 +1,1090 @@
-# PRE DRAFT repository for a coding style PER
+# Coding Style Guide
```
Comment (vdelau) [inline README.md:1]: Personally I'd like to separate the spec from the README. For me the README should be a quick introduction to the repo with pointers where to find more information. The spec should IMHO be a separate document.

**Comment by samdark** [comment]
@KorvinSzanto you mean pure PSR-12 text without **any** modifications?

File: README.md (hunk 2 above)
Comment (samdark) [reply README.md]: Done.

**Comment by KorvinSzanto** [comment]
> you mean pure PSR-12 text without **any** modifications?

Exactly, I am not too worried about the PSR / PER distinction but just for posterity sake I like the idea of every change being tracked. #2 is exactly what I was looking for, it should be able to merge prior to #1 merging without conflicts.

File: meta.md [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+| Alexander Makarov       |  Yii framework    | ✓ | ✓ | ✓ | ❌ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ |
+| Korvin Szanto           | concrete5         | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ |
+| Leo Feyer               | Contao            | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ |
+| Larry Garfield          | Drupal            | ✓ | ✓ | ✓ | ✓ | ✓ | ❌ | ✓ | ✓ | ✓ | ❌ | ✓ | ✓ | ❌ | ✓ | ✓ |
+| André R.                | eZ                | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ |
+| Jan Schneider           | Horde             | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ |
+| Karsten Dambekalns      | Neos and Flow     | ✓ | ✓ | ✓ | ✓ | ❌ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ |
+| Andres Gutierrez        | Phalcon           | ❌ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ |
```
Comment (Jeckerson) [inline meta.md:103]:
```suggestion
| Nikolaos Dimopoulos        | Phalcon           | ❌ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ |
```

https://www.php-fig.org/personnel/

Comment (samdark) [reply meta.md:103]: Hmm. Was representative the same at the time of PSR-12?

Comment (Jeckerson) [reply meta.md:103]: Only until PSR-12, now Nikolaos is current representative.

Comment (samdark) [reply meta.md:103]: But that table is about voting for feature back then.

Comment (Jeckerson) [reply meta.md:103]: OK

File: meta.md [hunk 4]
```diff
@@ -176,36 +176,34 @@ specification for a full understanding of its contents.
...
-* Cees-Jan Kiewiet for moral support
+* Everyone involved in PSR-1, PSR-2, PSR-12.
 
-# 7. Votes
+## 7. Votes
 
-* **Entrance Vote:** https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/php-fig/P9atZLOcUBM/_jwkvlYKEAAJ
-* **Approval Vote:** https://groups.google.com/forum/#!topic/php-fig/1uaeSMaDGbk
+* **Entrance Vote:** 
+* **Approval Vote:**
```
Comment (KorvinSzanto) [inline meta.md:203]: This should be updated with the PER approval vote

File: meta.md [hunk 5]
```diff
@@ -176,36 +176,34 @@ specification for a full understanding of its contents.
...
+* **Approval Vote:** 
 
-# 8. Relevant Links
+## 8. Relevant Links
 
 _**Note:** Order descending chronologically._
 
-* [Inspiration Mailing List Thread](https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!topic/php-fig/wh9avopSR9k)
-* [Initial Mailing List PSR Proposal Thread](https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!topic/php-fig/MkFacLdfGso)
+*
```
Comment (KorvinSzanto) [inline meta.md:209]: Might be good to include a link to PSR-12 here

File: spec.md [hunk 6]
```diff
@@ -8,24 +8,24 @@ interpreted as described in [RFC 2119][].
 
 ## Overview
 
-This specification extends, expands and replaces [PSR-2][], the coding style guide and
+This specification is a live document that extends, expands and replaces [PSR-12][], the extended coding style guide and
```
Comment (KorvinSzanto) [inline spec.md:11]: In my opinion this document is not a living document. The latest commit is not the source of truth the CC approved tag is.

File: meta.md [hunk 7]
```diff
@@ -176,36 +176,34 @@ specification for a full understanding of its contents.
...
-## 6.3. Working Group Members:
+### 6.3. Working Group Members:
 
-* Alessandro Lai
 * Alexander Makarov
-* Michael Cullum
-* Robert Deutz
+* 
+* 
+*
```
Comment (KorvinSzanto) [inline meta.md:194]: Probably want to get this list shored up now that we have an official WG

**Review by KorvinSzanto — APPROVED** [review]
I think this is good to merge but we should make a couple additional changes prior to a vote to tag 1.0.0 as noted in my comments

**Comment by samdark** [comment]
@KorvinSzanto I thought the idea was to tag PSR-12 original as 1.0.0 without any changes / fixes.

**Comment by KorvinSzanto** [comment]
That doesn't makes a whole lot of sense to me. I advocated for getting PSR-12 merged verbatim before this PR so that the git history would include all changes, I didn't mean to suggest that should also apply to the versioning. In my opinion a 1.0.0 should have proper metadata and not be labeled PSR-12 IMO. 

We should discuss more in discord.

File: meta.md (hunk 1 above)
Comment (Crell) [reply meta.md:87]: There's a lot of stuff from the meta doc we can probably just toss out at this point, and I'd honestly like to see the spec part split into multiple pages for ease of reading/editing.  That's all post-1.0 though.

File: spec.md (hunk 6 above)
Comment (samdark) [reply spec.md:11]: Makes sense. Will fix that.

**Pull request merged**